Mikado-Website/index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Mikado - UBC Launchpad</title>
+  <meta content="width=device-width, initial-scale=1.0" name="viewport">
+  <meta content="" name="keywords">
+  <meta content="" name="description">
+
+  <!-- Favicons -->
+  <link href="img/favicon.png" rel="icon">
+  <link href="img/apple-touch-icon.png" rel="apple-touch-icon">
+
+  <!-- Google Fonts -->
+  <link href="https://fonts.googleapis.com/css?family=Thasadith" rel="stylesheet"> 
+
+  <!-- Bootstrap CSS File -->
+  <link href="lib/bootstrap/css/bootstrap.min.css" rel="stylesheet">
+
+  <!-- Libraries CSS Files -->
+  <link href="lib/font-awesome/css/font-awesome.min.css" rel="stylesheet">
+  <link href="lib/fancybox/fancybox.css" rel="stylesheet">
+
+  <!-- Main Stylesheet File -->
+  <link href="css/style.css" rel="stylesheet">
+
+  <link rel="prefetch" href="img/zoom.png">
+
+
+</head>
+
+<body data-spy="scroll" data-offset="58" data-target="#navigation">
+
+  <!-- Fixed navbar -->
+  <div id="navigation" class="navbar navbar-fixed-top">
+    <div class="container">
+      <div class="navbar-header">
+        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+          <i class="fa fa-bars"></i>
+        </button>
+      </div>
+      <!--/.nav-collapse -->
+    </div>
+  </div>
+
+  <!-- === MAIN Background === -->
+  <div class="slide story" id="intro" data-slide="1">
+    <div class="container">
+      <div id="home-row-1" class="row clearfix">
+        <div class="col-12">
+          <h1 class="font-semibold">MIKADO </h1>
+          <h4 class="font-thin"> <span class="font-semibold">Organize </span> your next adventure.</h4>
+          <br>
+          <br>
+        </div>
+        <!-- /col-12 -->
+      </div>
+      <!-- /row -->
+      <!-- /row -->
+    </div>
+    <!-- /container -->
+  </div>
+  <!-- /slide1 -->
+
+  <!-- === Slide 2 === -->
+  <div class="slide story" id="about-us" data-slide="2">
+    <div class="container">
+      <div class="row title-row">
+        <div class="col-12 font-thin"> Scroll left or right to preview Mikado<br> </div>

```

Comment:
Don't need this text here, too large (could move below image right under scroll bar, and much smaller)